region/src/main/java/org/motechproject/nms/region/domain/State.java

```diff
@@ -80,22 +68,14 @@ public void setCode(Long code) {
         this.code = code;
     }
 
-    public List<District> getDistricts() {
+    public Set<District> getDistricts() {
```

Comment:
Since we're going to have to actually change the database (for the location schema change) maybe we could get rid of the column?